### PR TITLE
iOS Swap Tx Notification

### PR DESCRIPTION
**Pull request by dangeross**
Adds iOS handling of address_txs_confirmed notification message

### CODE REVIEW

File: android/app/src/main/kotlin/com/cBreez/client/job/LnurlPayInfo.kt [hunk 1]
```diff
@@ -58,16 +59,16 @@ class LnurlPayInfoJob(
...
+                fail(e.message, request.replyURL)
+            }
             notifyChannel(
                 context,
                 Constants.NOTIFICATION_CHANNEL_LNURL_PAY,
                 context.getString(R.string.lnurl_pay_notification_failure_title),
             )
         }
-
-        fgService.shutdown()
```
Comment (erdemyerebasmaz) [inline android/app/src/main/kotlin/com/cBreez/client/job/LnurlPayInfo.kt:71]: Why was this removed? Is it because `LnurlPayInvoice` task is expected to start soon afterwards?

What if we don't have enough inbound liquidity to pay the lnurl invoice? I don't think we do receive a _"lnurlpay_invoice"_ notification for rejected/failed cases.

Comment (dangeross) [reply android/app/src/main/kotlin/com/cBreez/client/job/LnurlPayInfo.kt]: My bad, sorry. Reverted the change

**Review by erdemyerebasmaz — APPROVED** [review]
LGTM
